src/Phaser.C

```diff
@@ -76,7 +82,7 @@ Phaser::out (float * smpsl, float * smpsr)
         rgain = 0.0f;
 
     for (i = 0; i < period; i++) {
-        float x = (float) i / fPERIOD;
+        float x = (float) i / float(period);
```

Comment:
Finally figured out why I did this type cast. Did the original conversion in December last year and forgot why I did it.  I probably copied from:
APhaser.C:106    invperiod = 1.0f / (float)PERIOD;//had to move this to run
From original rakarrack in init:
APhaser.C:79    invperiod = 1.0f / fPERIOD;
btw, fPERIOD was declared in APhaser.h but never used.

Personally, what was done in Pan in clearer to me:
Pan.C:68:    float fPERIOD = period;
or perhaps even clearer (and redundant) float fPERIOD = (float)period;

Whatever works for you :)!